Rubberduck.Parsing/Symbols/AccessibilityCheck.cs

```diff
@@ -4,94 +4,146 @@ public static class AccessibilityCheck
     {
         public static bool IsAccessible(Declaration callingProject, Declaration callingModule, Declaration callingParent, Declaration callee)
         {
-            if (callee.DeclarationType.HasFlag(DeclarationType.Project))
+            if (callee == null)
+            {
+                return false;
+            }
+            else if (callee.DeclarationType.HasFlag(DeclarationType.Project))
             {
                 return true;
             }
-            if (callee.DeclarationType.HasFlag(DeclarationType.Module))
+            else if (callee.DeclarationType.HasFlag(DeclarationType.Module))
```

Comment:
Also a superfluous else. we return if the previous expression was true, so there's no need for the else